mappings/net/minecraft/client/particle/Particle.mapping

```diff
@@ -20,17 +22,27 @@ CLASS net/minecraft/class_502 net/minecraft/client/particle/Particle
 		ARG 8 velocityX
 		ARG 10 velocityY
 		ARG 12 velocityZ
+	METHOD method_1282 setMiscTexture (I)V
+		ARG 1 textureIndex
 	METHOD method_1283 (Lnet/minecraft/class_520;Lnet/minecraft/class_864;FFFFFF)V
 		ARG 1 builder
 		ARG 2 entity
-	METHOD method_9698 setTexture (Lnet/minecraft/class_1639;)V
-		ARG 1 sprite
-	METHOD method_1286 getRed ()F
+	METHOD method_1283 render (Lnet/minecraft/class_533;FFFFFF)V
 	METHOD method_1285 setColor (FFF)V
 		ARG 1 red
 		ARG 2 green
 		ARG 3 blue
+	METHOD method_1286 getRed ()F
+	METHOD method_1287 scaleVelocity (F)Lnet/minecraft/class_502;
 	METHOD method_1288 getGreen ()F
+	METHOD method_1289 setScale (F)Lnet/minecraft/class_502;
+		ARG 1 scale
 	METHOD method_1290 getBlue ()F
+	METHOD method_4296 setAlpha (F)V
+		ARG 1 alpha
+	METHOD method_5132 setTex (Lnet/minecraft/class_1557;)V
```

Comment:
```suggestion
	METHOD method_5132 setTexture (Lnet/minecraft/class_1557;)V
```

Reply:
I called it that because thats what they call it in the stack trace, but okay ![abc](https://i.imgur.com/vWFtEbe.png)

Reply:
done in cfd2745